taxi/forms.py

```diff
@@ -50,3 +50,40 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Search by username"}
+        )
+    )
+
+
+class CarSearchForm(forms.Form):
+    model = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Search by model"}
+        )
+    )
+
+    class Meta:
+        model = Car
+        fields = ["model"]
```

Comment:
The 'Meta' class in 'CarSearchForm' is not necessary because 'forms.Form' does not use it. You should remove it to avoid confusion.